test/gtest/ucp/test_ucp_mmap.cc

```diff
@@ -623,21 +605,38 @@ void test_ucp_mmap::test_rereg_imported_mem(ucp_mem_h memh,
     pack_params.field_mask = UCP_MEMH_PACK_PARAM_FIELD_FLAGS;
     pack_params.flags      = memh_pack_flags;
 
-    status = ucp_memh_pack(memh, &pack_params, &exported_memh_buf,
+    status = ucp_memh_pack(exported_memh, &pack_params, &exported_memh_buf,
                            &exported_memh_buf_size);
     if ((status == UCS_ERR_UNSUPPORTED) &&
         (pack_params.flags & UCP_MEMH_PACK_FLAG_EXPORT)) {
         UCS_TEST_SKIP_R("memory exporting is unsupported");
     }
     ASSERT_UCS_OK(status);
 
-    ucp_mem_h imp_memh;
-    import_memh(exported_memh_buf, &imp_memh);
+    ucp_mem_map_params_t params;
+    params.field_mask           =
+            UCP_MEM_MAP_PARAM_FIELD_EXPORTED_MEMH_BUFFER;
+    params.exported_memh_buffer = exported_memh_buf;
 
-    ucp_mem_h test_imp_memh;
-    import_memh(exported_memh_buf, &test_imp_memh);
+    ucp_mem_h imported_memh;
+    scoped_log_handler warn_slh(import_no_md_error_handler);
+    status = ucp_mem_map(receiver().ucph(), &params, &imported_memh);
+    if (status == UCS_ERR_UNREACHABLE) {
+        release_exported_memh_buf(exported_memh_buf);
+        UCS_TEST_SKIP_R("memory importing is unsupported");
+    }
 
     release_exported_memh_buf(exported_memh_buf);
+    ASSERT_UCS_OK(status);
+    return imported_memh;
+}
+
+void test_ucp_mmap::test_rereg_imported_mem(ucp_mem_h memh,
+                                            uint64_t memh_pack_flags,
```

Comment:
seems `memh_pack_flags` is not really needed

Reply:
ok